src/App.js

```diff
@@ -27,7 +27,7 @@ export default function App() {
             <Route path="/portfolio">
               <Portfolio />
             </Route>
-            <Route path="/" component={() => <Home currency={currency} />} />
+            <Route path="/" component={(props) => <Home currency={currency} {...props} />} />
```

Comment:
mental note: you will have issues with page reloading and it will be fixed with redux